src/main/java/com/ericsson/eiffel/remrem/semantics/clone/CloneEiffelRepository.java

```diff
@@ -0,0 +1,42 @@
+package com.ericsson.eiffel.remrem.semantics.clone;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.jgit.api.Git;
+
+import com.ericsson.eiffel.remrem.semantics.eiffelSchemas.EiffelRepoJsonSchema;
+import com.ericsson.eiffel.remrem.semantics.eiffelSchemas.EiffelConstants;
+
+/**
+ * This class is used to Clone the Eiffel Repo from github
+ * 
+ * @author xdurvak
+ */
+
+public class CloneEiffelRepository {
+	private String homeDir = System.getProperty(EiffelConstants.USER_HOME);
+	private static File localPath;
+
+	private void cloneEiffelRepo(String repoURL, String branch) {
```

Comment:
This method is called cloneEiffelRepo but does all the conversion as well. I would move the last line to the main method. Then the method has a suitable name.